packages/sitecore-jss-react/src/components/Placeholder.test.tsx

```diff
@@ -505,8 +522,7 @@ it.skip('should not render Suspense when disableSuspense is true', () => {
     </SitecoreContext>
   );
 
-  expect(renderedComponent.container.querySelectorAll('ErrorBoundary').length).to.equal(1);
-  expect(renderedComponent.container.querySelectorAll('Suspense').length).to.equal(0);
+  expect(renderedComponent.container.innerHTML).to.not.contain('Loading component...');
```

Comment:
Same as above, can we verify that component is rendered with expected markup?
"No error"